src/main/java/controller/AttendanceCheckController.java

```diff
@@ -0,0 +1,72 @@
+package controller;
+
+import domain.Attendance;
+import domain.AttendanceCustomDate;
+import domain.CrewAttendances;
+import domain.Month;
+import exception.CrewNotExistException;
+import exception.DuplicateAttendanceException;
+import view.InputView;
+import view.OutputView;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class AttendanceCheckController implements Controller {
+    private final InputView inputView;
+    private final OutputView outputView;
+    private final CrewAttendances crewAttendances;
+
+    public AttendanceCheckController(
+            InputView inputView,
+            OutputView outputView,
+            CrewAttendances crewAttendances
+    ) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.crewAttendances = crewAttendances;
+    }
+
+    @Override
+    public void run() {
+        LocalDate now = AttendanceCustomDate.now().toLocalDate();
+        validateDate(now);
+        String name = inputView.readName();
+        String timeInput = inputView.readTime();
+        String[] minuteAndHour = timeInput.split(":");
+        LocalDateTime dateTime = LocalDateTime.of(
+                now.getYear(),
+                now.getMonthValue(),
+                now.getDayOfMonth(),
+                Integer.parseInt(minuteAndHour[0]),
+                Integer.parseInt(minuteAndHour[1])
+        );
```

Comment:
```suggestion
        String timeInput = inputView.readTime();
        int hour = Integer.parseInt(timeInput.split(":")[0]);
        int minute = Integer.parseInt(timeInput.split(":")[1]);
        LocalDateTime dateTime = LocalDateTime.of(now, LocalTime.of(hour, minute));
```

개인적인 취향인데 위와 같이 LocalDateTime.of(LocalDate, LocalTime)을 사용하면 더 간결하게 LocalDateTime을 만들 수 있을 것 같아요.